.code-samples.meilisearch.yaml

```
@@ -758,3 +764,69 @@ tenant_token_guide_search_no_sdk_1: |-
   curl \
     -X POST 'http://127.0.0.1:7700/indexes/patient_medical_records/search' \
     -H 'Authorization: Bearer yourTenantToken'
+get_typo_tolerance: |-
+  curl \
+    -X GET 'http://localhost:7700/indexes/books/settings/typo-tolerance'
+update_typo_tolerance: |-
+  curl \
+    -X POST 'http://localhost:7700/indexes/books/settings/typo-tolerance' \
+    -H 'Content-Type: application/json' \
+    --data-binary '{
+      "minWordSizeForTypos": {
+        "oneTypo": 4,
+        "twoTypos": 10
+      },
+      "disableOnAttributes": [
+        "title"
+    ]
+      }'
+reset_typo_tolerance: |-
```

```suggestion
get_typo_tolerance_1: |-
  curl \
    -X GET 'http://localhost:7700/indexes/books/settings/typo-tolerance'
update_typo_tolerance_1: |-
  curl \
    -X POST 'http://localhost:7700/indexes/books/settings/typo-tolerance' \
    -H 'Content-Type: application/json' \
    --data-binary '{
      "minWordSizeForTypos": {
        "oneTypo": 4,
        "twoTypos": 10
      },
      "disableOnAttributes": [
        "title"
    ]
      }'
reset_typo_tolerance_1: |-
```

the `_1` were missing. If you dropped this convention please ignore my suggestion!